src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,12 @@
+package duke;
+public class Todo extends Task {
+
+    public Todo(String description) {
```

Comment:
Perhaps need a descriptive header comments for the Todo method? 🤔
And I noticed the same issue in several other places below.